usr/sbin/rear

```diff
@@ -236,7 +236,7 @@ ARGS=( "$@" )
 
 # The following workflows are always verbose:
 case "$WORKFLOW" in
-    (validate|shell|recover)
+    (validate|shell|recover|mountonly)
```

Comment:
Only a side note:
Now I wonder if also the `restoreonly` workflow should be always verbose?
My reasoning is when `recover` is verbose then its backup restore part runs verbose
so that the backup restore part alone (via `restoreonly`) should also run verbose.
In general I wonder if all workflows that are run from inside of the booted
ReaR recovery system should be always verbose?

Reply:
FWIW, I agree with your proposal, but I've not implemented it in the current branch as I considered out of scope.